added simplified WP diagram

RachelComerford opened the pull request:
<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/RachelComerford/wpub/pull/112.html" title="Last updated on Dec 2, 2017, 4:52 PM GMT (07f977c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wpub/083a783...RachelComerford:07f977c.html" title="Last updated on Dec 2, 2017, 4:52 PM GMT (07f977c)">Diff</a>

iherman commented:
@RachelComerford some minor remarks,

- The alt text contains the term "things". This should be 'things', because the " sign is used to delimit the attribute.
- You should also upload the svg file itself, it does not seem to be in the repo... actually, it is probably a good idea to put it into `common/images` rather than on the top level, just to keep in line with the way the repository is organized

@mattgarrish would we prefer to use the `<figure>` element with a `<caption>` to encapsulate  the image? It may be better...

TzviyaSiegman commented:
@mattgarrish `<caption>` or `<figcaption>`?

iherman commented:
> On 1 Dec 2017, at 16:55, Tzviya <notifications@github.com <mailto:notifications@github.com>> wrote:
> 
> @mattgarrish <https://github.com/mattgarrish> <caption> or <figcaption>?
> 
> 

sorry, figcaption

> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/w3c/wpub/pull/112#issuecomment-348531418>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAfyE8XVCgX7b3ZOjL_9RzfyO-v_AhEJks5s8CF3gaJpZM4Qyfza>.
> 
>  <https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png> <https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png> <https://github.com/w3c/wpub> <https://github.com/w3c/wpub/pull/112#issuecomment-348531418>

----
Ivan Herman, W3C
Publishing@W3C Technical Lead
Home: http://www.w3.org/People/Ivan/ <http://www.w3.org/People/Ivan/>
mobile: +31-641044153
ORCID ID: http://orcid.org/0000-0003-0782-2704 <http://orcid.org/0000-0003-0782-2704>

mattgarrish commented:
> common/images

I'd suggest an image directory outside of common, as common was intended to be for shareable resources across WP/PWP/EPUB4 and anything else.

RachelComerford commented:
updated with figure and figcaption, changed 'things', added the image to the repo, and taught my team a bunch of new curse words while learning how to git from the command line 😄

tobie commented:
Fixed the preview and diff links. Sorry for the delay in getting it generated.

mattgarrish commented:
Please use relative paths for referencing or else this becomes a permanent link to your clone.

And, again, it doesn't belong in /common. Create a /images directory in the root so that we keep resources that are actually shared separate.

iherman commented:
@tobie thanks!

mattgarrish commented:
The previews aren't showing your SVG as there's an access denied error getting the URL, but it can be viewed here:
https://rawgit.com/RachelComerford/wpub/master/images/WP-diagram-new.svg

mattgarrish commented:
FYI, there's a typo in the alt text that needs attention: imclides

Also, from an accessibility perspective, it would be helpful to put the description into a details/summary element (or something similar). It's very long to be the alt text. You can probably use an empty alt attribute on the image as any alternative would probably be redundant with the existing figcaption.

iherman commented:
Closed this without merge in favour of #113. The main difference is in the tool used for the diagram.
